htdocs/install/mysql/tables/llx_bookmark.sql

```diff
@@ -21,7 +21,7 @@ create table llx_bookmark
   rowid       integer AUTO_INCREMENT PRIMARY KEY,
   fk_user     integer NOT NULL,
   dateb       datetime,
-  url         varchar(255) NOT NULL,
+  url         varchar(512) NOT NULL,
```

Comment:
Because there is a unique key on this field, and because index need 3 bytes per char, and because mysql does not allow index larger than 768 bytes, using 512 may generated errors when you insert data with 500 char length (500 x 3 > 768)
Don't you experience such a trouble ? Which database do you use ?

Reply:
Why is there a unique key on the URL column ? Meaning two users can't save the same bookmark ?

Reply:
The unique key is on couple
url, fk_user.

ALTER TABLE llx_bookmark ADD UNIQUE uk_bookmark_url   (fk_user, url);

It means a user can't have duplicate bookmarks.